src/languages/es.js

```diff
@@ -368,7 +369,6 @@ export default {
     tabSelector: {
         manual: 'Manual',
```

Comment:
@arosiclair Manual is not translated. Do we want to fix this part of this PR?
Should be a simple change IMO

Reply:
"Manual" is the translation 😄 . It's the same in English and Spanish

Reply:
OOPS, didn't know that. Seems i am pretty weak in translations 🫣. 
Code is working fine AFAIK. I will finish this review in couple of hours.